scripts/_common.sh

```diff
@@ -96,6 +93,52 @@ ynh_remove_systemd_socket_config () {
     fi
 }
 
+_ynh_add_dnsmasq() {
+    # Declare an array to define the options of this helper.
+    local legacy_args=t
+    local -A args_array=( [t]=template= )
+    local template
+    ynh_handle_getopts_args "$@"
+    local template="${template:-dnsmasq.conf}"
+
+    ynh_add_config --template="$template" --destination="/etc/dnsmasq.d/$app"
```

Comment:
```suggestion
    ynh_add_config --template="dnsmasq.conf" --destination="/etc/dnsmasq.d/$app"
```

Let's remove the getops part